sdks/python/apache_beam/utils/counters.py

```diff
@@ -67,7 +67,7 @@ def __new__(cls, name, stage_name=None, step_name=None,
                                            origin, output_index, io_target)
 
   def __str__(self):
-    return '%s' % self._str_internal()
+    return '<CounterName<%s> at %s>' % (self._str_internal(), hex(id(self)))
```

Comment:
`__str__` defaults to `__repr__`, if you want them to be the same remove this one.